### PR TITLE
CSS for registration

### PR DESCRIPTION
**Pull request by stefanbretzel**
I modified the CSS pertaining the registration form. A new css class media_registration is introduced that can be applied to a div. It modifies the behaviour of the div (and the iframe below it) so that the iframe and its contents also behaves in a responsive way (i.e. the registration form goes from two to one-column). In addition, two CSS media queries are added that hide the iframe and offer a link to the registration form if the screen width goes below 500px.

The two changes are fairly independent, i.e. removing the @media instructions in index.html will disable the hiding of the iframe/showing the link instead while the registration_form class in the CSS govern the sizing and behaviour of the iframe. I chose the height of 1400px for the iframe within the meeting_registration div is chosen "heuristically", so that the whole page is shown including footer etc.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,32 @@
...
+.meeting_registration {
+	display: flex;
+	width: 100%;
+	height: 100%;
+	flex-direction: column;
+	overflow: hidden;
+}  
+
+div.meeting_registration > iframe {
+	height:1400px;
```
Comment (akhmerov) [inline index.html:23]: I've cut it at a lower height intentionally to not show the footer.

```suggestion
	height:910px;
```

Comment (akhmerov) [reply index.html:23]: This doesn't work when the iframe switches to a single column layout though :(

Comment (stefanbretzel) [reply index.html:23]: What's "this"? Your new height or something else? (and using what browser)

When it's the height, the general problem is that you can't determine the height of the page included in the iframe with CSS only. So in the end you experimentally have to find the right value (or live with an unfavorably cropped page).

**Comment by akhmerov** [comment]
I see the registration link when using a smartphone, but not in a very narrow window of a desktop browser, do you know why? Is there perhaps a range of widths where both are hidden?

**Comment by stefanbretzel** [comment]
Ok, strange. In theory the iframe should be hidden once the width gets below 500px and the link is hidden if the width exceeds 500px. So what I'd expect in the "worst case" is that you'll see both when you exactly hit a width of 500px. Why you see neither, I currently do not understand. I've tested the same thing (making the window narrower and narrower) with FF 75 on ubuntu and the form was hidden and the link appeared as intended. What browser are you using?

**Comment by akhmerov** [comment]
Ah! I understand! Chrome doesn't allow shrinking itself below *exactly* 500px. That's also the only width where neither condition is fulfilled due to one-off error.

File: index.html (hunk 1 above)
Comment (akhmerov) [reply index.html:23]: Sorry, I indeed meant the height. I am aware that the iframe contents are not accessible.

The `910` value I indeed determined by trial and error, but it only works for two column layout (which is probably more common using screens).

**Comment by akhmerov** [comment]
I propose to change the transition width to `1098`—this is where zoom decides to rearrange its form into a single column. That way we may keep `910` as height and avoid the nasty footer.

**Comment by akhmerov** [comment]
Update here: we now have control over the zoom side css.

I used it to remove the footer so far, but now we should be utilize it much easier.

**Comment by akhmerov** [comment]
We are likely going to use an approach similar to #149 here

**Pull request closed without merging**